source/js/buyers-guide/product-tab.js

```diff
@@ -22,12 +25,20 @@ function scrollTabButtons(tab, position) {
 }
 
 function switchToTab(tab, position) {
+  let contentContainer;
+
   removeHighlight(currentTab);
   currentTab = setHighlight(tab);
   scrollTabButtons(tab, position);
-  const contentContainer = document.querySelector(
-    `div[data-product-label="${position}"`
-  );
+
+  content.querySelectorAll(`div[data-product-label]`).forEach((container) => {
```

Comment:
We need to conditionally change CSS visibility of each tab's content. Only the current active tab should have its content visible (in CSS). Otherwise tab order will go from the active tab content to the next tab's content (outside of the viewport)